### ISSUE TITLE
[BUG] Flightmodel - AoA & Trimming

### ISSUE THREAD

**Issue by Funda86** [flybywiresim/a32nx#2954]
**Is your feature request related to a problem? Please describe.**

At cruise level the AoA and the trimming looks a bit unrealistic and the nose is a bit too high, expecially when you are flying with a full loaded plane

**Describe the feature you'd like to see implemented**

I've tried to configure the flight_model.cfg and I realized that with a cruise_lift_scalar value of 1.3 (instead of 1 - in FLIGHT_TUNING) the issue is quite solved and the plane responds better 

**References**

 I don't have any reference regarding this

**Additional context**

Discord username (if different from GitHub):

David Funda #6596

**Comment by MisterChocker** [comment]
cc. @donstim

@Funda86 next time please use the bug template for reporting issues or bugs.
Please provide the contents from the build_info.json.

**Comment by donstim** [comment]
We strive for the utmost in realism. The current flight model matches the real airplane pitch angles in cruise. What pitch  angle were you seeing and how did you determine it is too high?

**Comment by Funda86** [comment]
> We strive for the utmost in realism. The current flight model matches the real airplane pitch angles in cruise. What pitch angle were you seeing and how did you determine it is too high?

I use to fly at FL360 with a loaded plane of about 80-90% in weight. The picth angle is at 3° and the trim behond 20% up... I have no specific notion on that but I used to watch a lot of video of a320 cockpit in cruise and the line of the horizon is not as high as in the Sim. Of course the higher you get the higher the pitch angle get. With the setting I suggested the angle rarely reach 3° and the trim stayes at around 15% (with full loaded plane). Looks better to me, but of course I could be wrong

**Comment by donstim** [comment]
The pitch angle will depend on your airspeed, altitude, and weight. For example, at FL350, 70T (155,000 lb), and 255 KCAS, the pitch angle should be 3 degrees. If your pitch angle rarely reaches 3 degrees during cruise, then it does not match the real airplane. If that's what you are comfortable with, then fine use it. But for us to make a change, we need to be presented with reference data showing that what we currently have does not match the real airplane.

**Comment by MisterChocker** [comment]
Please provide reference

**Comment by Funda86** [comment]
![TFXZomN](https://user-images.githubusercontent.com/73944217/104122033-ea5f6780-5342-11eb-957b-5978fbb97e3d.png)

This is what I found as reference. This should be the desired pitch attitude for an A320. This come from someone who posted it in a forum on this specific topic.

**Comment by donstim** [comment]
Thank you for providing a reference for your issue. That is the type of thing we need to evaluate any issue like yours.

The table you provided does not contain any identifying information as to what specific airplane model it pertains to. There is only the forum post (https://forums.flightsimulator.com/t/a320-angle-of-attack-at-cruise/192090/24?u=awareplot117729) where the poster claims it is for "A320." We have numerous flight documents for the the A318/A319/A320/A321 series, and this particular chart does not match the unreliable airspeed charts in any of the documents we have.

We conducted an extensive set of tests to ensure the A32NX matches the unreliable airspeed pitch angle charts for the A320NEO airplane that is being modeled. See https://github.com/flybywiresim/a32nx/issues/2954. The data that we used is documented there. We did have to make compromises on the pitch attitude for some configurations because of the way that MSFS models slat lift, but the flaps up configuration was not one of them.  The pitch attitude in cruise matches the data and was confirmed correct by real life A320NEO pilots.
